baselines/aad.md

```diff
@@ -649,40 +629,40 @@ Performing a manual review of highly privileged users to determine which ones ar
     Get-MgBetaUser -Filter "userPrincipalName eq 'username@somedomain.com'" | FL
```

Comment:
This displayed field is inconsistent with word document (which has an additional middle line with Select-Mgprofile beta)

Reply:
Fixed in the Word doc which was wrong. They match now.